pkg/logbridge/doc.go

```diff
@@ -0,0 +1,12 @@
+package logbridge
+
+/*
```

Comment:
Package-level documentation needs to go _above_ the `package` declaration for [godoc](http://blog.golang.org/godoc-documenting-go-code) to recognize it.